habitat-lab/habitat/sims/habitat_simulator/habitat_simulator.py

```diff
@@ -281,7 +282,11 @@ def __init__(self, config: DictConfig) -> None:
         for path in self.habitat_config.additional_object_paths:
             obj_attr_mgr.load_configs(path)
         self._action_space = spaces.Discrete(
-            len(self.sim_config.agents[0].action_space)
+            len(
+                self.sim_config.agents[
+                    self.habitat_config.default_agent_id
+                ].action_space
```

Comment:
Use `get_agent_config` for consistency:
```suggestion
            len(
                get_agent_config(self.sim_config).action_space
```

Reply:
Here `self.sim_config` is not of type `DictConfig` it is of type `habitat_sim.Configuration`. So the code should be kept as it is.